### DISCUSSION TITLE
Make Scratch  Addons compatible with mobile Firefox

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#3255] [Ideas] [answered]
I'm pretty sure this won't happen, but I thought I'd post my idea here anyway.

**Comment by WorldLanguages** [comment]
Why isn't it compatible already?

**Reply by Samq64** [reply]
I don't know. The `Add to Firefox` button is just grayed out for some reason with a notice under it saying "To find add-ons compatible with Firefox for Android, click here."

**Reply by mxmou** [reply]
It's probably already technically compatible, but only recommended extensions are currently supported.

**Comment by TheColaber** [comment]
Sorry, but I don't think this is possible, as they basically only allow extensions with tons of users.
![image](https://user-images.githubusercontent.com/72760579/128584953-e2d4dcfa-89db-49ef-812d-308a3ae167d1.png)

**Comment by pufferfish101007** [comment, marked as answer]
You can get firefox nightly and follow the instructions [here](https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/). A couple of addons don't work very well (searchable-dropdowns 👀), but most do work.
